react/src/reactTextMask.js

```diff
@@ -1,41 +1,26 @@
-import React, {PropTypes} from 'react'
-import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
-
-export const MaskedInput = React.createClass({
-  propTypes: {
-    mask: PropTypes.oneOfType([
-      PropTypes.array,
-      PropTypes.func,
-      PropTypes.bool,
-      PropTypes.shape({
-        mask: PropTypes.oneOfType([PropTypes.array, PropTypes.func]),
-        pipe: PropTypes.func
-      })
-    ]).isRequired,
-    guide: PropTypes.bool,
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    pipe: PropTypes.func,
-    placeholderChar: PropTypes.string,
-    keepCharPositions: PropTypes.bool,
-    showMask: PropTypes.bool
-  },
-
-  createTextMaskInputElement,
```

Comment:
also.. [this test](https://github.com/text-mask/text-mask/blob/master/react/test/reactTextMask.spec.js#L89) and [this test](https://github.com/text-mask/text-mask/blob/master/react/test/reactTextMask.spec.js#L118) relying on being able to stub the `createTextMaskInputElement` method - currently they are not failing because the method is not being called on the component class.. again, it is not needed to make the component function.. but is there to aid testing.

It might be that there is a better way to test this.. (ideas/PRs welcome).. but I think this particular PR should not make those changes if it doesn't need to.